dataquality/integrations/torch.py

```diff
@@ -229,6 +232,12 @@ def watch(
     :param model: Pytorch Model to be wrapped
     :param dataloaders: List of dataloaders to be wrapped
     :param last_hidden_state_layer: Layer to extract the embeddings from
+    :param unpatch_on_start: Force unpatching of dataloaders
+        instead of global patching
+    :param allow_missing_ids: Allow missing ids in the dataset if a RandomSampler
+        or WeightedRandomSampler is used. This is useful when logging less samples
+        during training than the total number of samples in the dataset.
```

Comment:
if this is user facing, this should be a little more explicit
```suggestion
    :param dataloader_random_sampling: Whether a RandomSampler
        or WeightedRandomSampler is being used. If random sampling is being used, you must set this to True, otherwise logging will fail at the end of training.
```